src/globalStyles/global.scss

```diff
@@ -122,9 +122,10 @@ th.reactable-header-sort-asc:after {
 }
 
 .table tbody tr td {
-  padding:6px 10px !important;
+  padding:3px 10px !important;
   border-top:none;
 }
 
-
-
+.table {
+  margin-bottom: 5px;
+}
```

Comment:
We already have some global css for table components: `.table tbody tr td`

I did the modification in the same file for consistency. Actually this style also applies to the tooltip tables as well. For example, cosmic tooltip had unnecessary white-space below the table.